pep-0597.rst

```diff
@@ -33,135 +34,140 @@ Using the default encoding is a common mistake
-It creates bugs that happen only on Windows. See [3]_, [4]_, [5]_,
+Even Python experts may assume that the default encoding is UTF-8.
+This creates bugs that only happen on Windows; see [3]_, [4]_, [5]_,
 and [6]_ for example.
 
-Emitting a warning when the ``encoding`` option is omitted will help
-to find such mistakes.
+Emitting a warning when the ``encoding`` argument is omitted will help
+find such mistakes.
 
 
 Explicit way to use locale-specific encoding
 --------------------------------------------
 
 ``open(filename)`` isn't explicit about which encoding is expected:
 
-* Expects ASCII (not a bug, but inefficient on Windows)
-* Expects UTF-8 (bug or platform-specific script)
-* Expects the locale encoding.
+* If ASCII is assumed, this isn't a bug, but is inefficient on Windows
+* If UTF-8 is assumed, this may be a bug or a platform-specific script
+* If the locale encoding is assumed, the behavior is as expected
+  (but could change if future versions of Python modify the default)
 
-In this point of view, ``open(filename)`` is not readable.
+From this point of view, ``open(filename)`` is not readable code.
 
 ``encoding=locale.getpreferredencoding(False)`` can be used to
-specify the locale encoding explicitly. But it is too long and easy
-to misuse. (e.g. forget to pass ``False`` to its parameter)
+specify the locale encoding explicitly, but it is too long and easy
+to misuse (e.g. one can forget to pass ``False`` as its argument).
 
 This PEP provides an explicit way to specify the locale encoding.
 
 
 Prepare to change the default encoding to UTF-8
 -----------------------------------------------
 
-Since UTF-8 becomes de-facto standard text encoding, we might change
-the default text encoding to UTF-8 in the future.
+Since UTF-8 has become the de-facto standard text encoding,
+we might default to it for opening files in the future.
 
-But this change will affect many applications and libraries. If we
-start emitting ``DeprecationWarning`` everywhere ``encoding`` option
-is omitted, it will be too noisy and painful.
+However, such a change will affect many applications and libraries.
+If we start emitting ``DeprecationWarning`` everywhere the ``encoding``
+argument is omitted, it will be too noisy and painful.
 
-Although this PEP doesn't propose to change the default encoding,
-this PEP will help the change:
+Although this PEP doesn't propose changing the default encoding,
+it will help enable that change by:
 
-* Reduce the number of omitted ``encoding`` options in many libraries
-  before we start emitting the ``DeprecationWarning`` by default.
+* Reducing the number of omitted ``encoding`` arguments in libraries
+  before we start emitting a ``DeprecationWarning`` by default.
 
-* Users will be able to use ``encoding="locale"`` option to suppress
-  the warning without dropping Python 3.10 support.
+* Allowing users to pass ``encoding="locale"`` to suppress
+  the warning, starting with Python 3.10.
```

Comment:
I wasn't 100% sure what you meant here by "dropping Python 3.10 support", especially since its not even released yet. I'm guessing you were referring to a hypothetical future version that didn't default to the locale encoding? Not 100% sure how this would change that...?

Reply:
> I'm guessing you were referring to a hypothetical future version that didn't default to the locale encoding?

I meant hypothetical future version that emits DeprecationWarning by default, not opt-in for using default encoding.
If we don't add `encoding="locale"` in 3.10, it is difficult to suppress the DeprecationWarning without breaking Python 3.10+ compatibility.

Reply:
Understood, thanks. What about this?

```suggestion
  the current warning and any `DeprecationWarning` added in the future,
  as well as retaining consistent behavior if later Python versions
  change the default, ensuring support for any Python version >=3.10.
```

Reply:
LGTM, but use ``` ``DeprecationWarning`` ```

Reply:
```suggestion
  the current warning and any ``DeprecationWarning`` added in the future,
  as well as retaining consistent behavior if later Python versions
  change the default, ensuring support for any Python version >=3.10.
```

Oops, good catch, thanks! Funny enough I added some additional checks for that in PR #1886 , but I made the same mistake myself, ha!